CHANGELOG.md

```diff
@@ -1,21 +1,113 @@
+1.0.0-RC / 2020-08-XX
+==================
+
+### Release notes
+
+This is a release candidate for 1.0.0 version. The goal of this RC is to collect feedback from users
+and provide 1.0.0 release with bugfixes and improvements based on that feedback.
+
+While preparing 1.0.0 version, we carefully examined public API of the library and split it in stable part,
+that we promise to be binary-compatible, and experimental declarations, that may change design in future.
+Experimental API is annotated with `@ExperimentalSerializationApi` annotation which requires opt-in.
+Also, significant part of the public API declarations was renamed to better reflect their purposes.
+
+The id of core artifact with `@Serializable` annotation and `Json` format was changed
+to be more clear and aligned with other kotlinx libraries.
+Please use `kotlinx-serialization-core` instead of `kotlinx-serialization-runtime`.
+If you have further questions, see [migration guide](TODO).
+
+#### Declarations deprecations and renaming
+
+* Rename core functions in StringFormat and BinaryFormat
+    * stringify/parse renamed to `encodeToString`/`decodeToString`
+    * Similarly, `encodeToByteArray/encodeToHexString/decodeFromByteArray/decodeFromHexString` in BinaryFormat should be used
```

Comment:
```suggestion
    * Similarly, `encodeToByteArray/encodeToHexString/decodeFromByteArray/decodeFromHexString` in BinaryFormat should be used instead of `dump`/`dumps`/`load`/`loads`
```